src/scss/components/_sidenav.scss

```diff
@@ -34,3 +30,14 @@
   padding-top: units(1.5);
   padding-bottom: units(1.5);
 }
+
+.usa-sidenav__sublist .usa-current {
+  @include remove-bar;
+  background-color: color('white');
+  color: $theme-color-primary-dark;
+  font-weight: bold;
```

Comment:
We might not need these anymore? It looks like with your other recent changes, these styles are already effectively inherited from styles defined above.

```suggestion
```